src/sentry/migrations/0078_case_insensitive_email.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-05-18 17:36
+from __future__ import unicode_literals
+
+from django.db import migrations
+from collections import defaultdict
+import sentry.db.models.fields.citext
+
+
+def dedeupe_emails(email_model):
```

Comment:
nit: but in my experience it's confusing to represent a model here as a lowercase name. I'd suggest it be preferable to shadow the name.

```python
def dedupe_emails(UserEmail):
```

Then everything else reads as normal.